packages/notification/src/__snapshots__/InlineNotification.spec.tsx.snap

```diff
@@ -55,7 +55,7 @@ exports[`<InlineNotification /> renders a basic notification correctly 1`] = `
 
 exports[`<InlineNotification /> renders a persistent notifications 1`] = `
```

Comment:
We'd prefer if you remove the snapshot test rather than fix them, as we've moved away from this pattern